compiler/install

```diff
@@ -65,6 +65,6 @@ do
   cp $jar $dest/$tool.jar
   echo '#!/bin/sh
 
-  java -jar '$dest'/'$tool'.jar $@' > $dest/$tool
+  java -jar '$dest'/'$tool'.jar "$@"' > $dest/$tool
```

Comment:
See my comments on #104. I'll accept this change, but I'm skeptical that it fixes anything. It seems to invite more problems/issues than it solves.